eclipse-collections-code-generator/src/main/resources/api/set/immutablePrimitiveSet.stg

```diff
@@ -61,7 +61,11 @@ public interface Immutable<name>Set extends Immutable<name>Collection, <name>Set
 
     /**
      * Returns the set of all objects that are members of both {@code this} and {@code set}. The intersection of
-     * [1, 2, 3] and [2, 3, 4] is the set [2, 3].
+     * [1, 2, 3] and [2, 3, 4] is the set [2, 3]. The output will contain instances from the smaller of the two sets
```

Comment:
primitives only have one sense of equality. They are value types, so there is no "memory reference" distinction. Ditto for the other 2 primitive stg files.

Reply:
I apologize for the oversight in my previous commit. I have now reverted these 3 primitive stg files.